src/mam4xx/aero_config.hpp

```diff
@@ -346,6 +346,68 @@ class Diagnostics final {
   // Ice Number Concentration due to imm freezing by bc in Mixed Clouds during
   // 10-s period [#/m3]
   ColumnView numimm10sbc;
+
+  // Values consumed by the convective processes to be filled upstream.
```

Comment:
Yes, I can beef up and hilight the comments about input/outputs to this process.  Is there a confluence page for documenting inputs/outputs of a process and maybe some notes about TODOs?

Reply:
I'm not sure we have a Confluence page that goes into any detail about process input/output. I think that would be lovely, but it would probably take quite a bit of effort to construct such a thing without the help of some automated tool such as Doxygen, we've used to generate [call graphs](https://eagles-project.atlassian.net/wiki/spaces/Computation/pages/211222689/Generating+a+Call+Graph+for+CAM+with+a+Representative+MAM4+Case+in+E3SMv1) in the past.

My above comment was mostly about leading the convproc-related diagnostics with some sort of "HEY THERE, READER!" markup in the `Diagnostics` class itself, rather like what you already have, but maybe with some attention getting kibble, like

```
  //=========================
  // Convective Process Diagnostics
  //=========================
```

Just a thought. You've included the information needed by the reader to understand the scope of the convproc diagnostic views already.